src/planscape/planning/views_v2.py

```diff
@@ -181,6 +182,7 @@ class ScenarioViewSet(MultiSerializerMixin, viewsets.ModelViewSet):
         "list": ListScenarioSerializer,
         "create": CreateScenarioV2Serializer,
         "retrieve": ScenarioV2Serializer,
+        "partial_update": UpsertConfigurationV2Serializer,
```

Comment:
Considering that `CreateScenarioV2Serializer` was renamed to `UpsertConfigurationV2Serializer`
```suggestion
        "create": UpsertConfigurationV2Serializer,
        "retrieve": ScenarioV2Serializer,
        "partial_update": UpsertConfigurationV2Serializer,
```